test/frontend_test.dart

```diff
@@ -0,0 +1,457 @@
+            .findElements(const By.cssSelector(
+                'video-summary-list > div.training-type-list > div '
+                '> video-summary > div.video-summary'))
+            .length;
+        await waitForAngular(driver);
+        print("Videos Found: " + videoItems.toString());
+        expect(await isXAtLeastValueOfY(videoItems, 6), true);
+      });
+    });
+
+    group("workouts page tests | formatorna light |", () {
+      setUpAll(() async {
+        await driver.get(url + "video/view/formatorna-light");
+        await waitForAngular(driver);
+        await sleep5();
+      });
+
+      test('at least 5 element rows under hol erheto el', () async {
+        var videosLength = await driver
+            .findElements(const By.xpath(
+                '*//workout-video-list/table/tbody/tr/td[1]/tag-representation/span'))
+            .length;
+        await waitForAngular(driver);
+        print("Available Programs: " + videosLength.toString());
+        expect(await isXAtLeastValueOfY(videosLength, 5), true);
+      });
+
+      test('at least 1 videotar under hol erheto el', () async {
+        var videotarOccurrences = await driver
+            .findElements(const By.xpath(
+                '*//workout-video-list/table/tbody/tr/td/a[contains(@href, "video")]'))
+            .length;
+        await waitForAngular(driver);
+        print("Occurrences of videotar: " + videotarOccurrences.toString());
+        expect(await isXAtLeastValueOfY(videotarOccurrences, 1), true);
+      });
+
+      test('content box on left of page has data next to labels', () async {
+        var descLinesLength = await driver
+            .findElements(
+                const By.xpath('*//video-short-description/div/table/tbody/tr'))
+            .length;
+        var valuesLength = await driver
+            .findElements(
+                const By.xpath('*//video-short-description/div/table/tbody'
+                    '/tr/td[2]'))
+            .length; //[. !='']
+        await waitForAngular(driver);
+        expect(await isXAtLeastValueOfY(valuesLength, descLinesLength), true);
+      });
+    });
+
+    group('scores page tests |', () {
+      setUpAll(() async {
+        await waitForAngular(driver);
+        await loginUserWithCredentials(
+            oldUserEmail, allUserPassword, driver, true);
+        await waitForAngular(driver);
+        await sleep2();
+        await driver.get(url + "profile/scores");
+        await waitForAngular(driver);
+        await sleep2();
+      });
+
+      test('user shows on highscore lists after login', () async {
+        var userScore =
+            await driver.findElement(const By.xpath('*//scores-list/div'
+                '/div[1]/div/ol/li/span[contains(text(), "connor")]'));
+        expect(await elementExists(userScore), true);
+      });
+
+      test('birth-List tab has data that includes users name', () async {
+        var birthListTab = await driver.findElement(const By.xpath(
```

Comment:
What is "birth-list" tab? Are you referring to "Örökranglista"? This means "All time scores".